docs/training/projects/project-during.md

```diff
@@ -1,42 +1,83 @@
+
+### Step 1: Personal Reflection and Goal Setting
+
+It might seem counterintuitive, but we think it works well to start your time together with a few minutes of personal reflection. The purpose of this time is for each project member to get clear on their personal project and learning goals, and to briefly communicate that in writing to their teammates.  
+
+We ask the project leader distribute a 4 x 6 card and writing utensil to each team member and then have each person reflect on the following for 3 minutes (set a timer!):
+
+```{admonition} PERSONAL CHECK-IN
+* What are my learning goals for the week?
+* How do I work best?
+* What skills / background am I bringing to this work?
+* Why is this work important to me?
+* What conditions do I need to thrive?
+* How will I know I've had a successful hackweek?
+```
+Next, each person should take another 3 minutes to write some of this information on your card. Start with your name and preferred pronouns, if you wish. Then include other information from your reflections above. Get creative, and don't worry about capturing everything. Use words or sketches as you like!
+
+Finally, give each member of your team up to 1 minute each (keep it short!) to share their name and a couple of items from their card. Be sure everyone has a chance to share. When you're done, keep your card in an accessible, readable location on the table so that others can look at it throughout the afternoon.
+
+### Step 2: Group Norms and Process
+
+Next, let's have a 10 - 15 minute conversation (depending on team size) about the group culture we hope to create during the week. 
+
+```{admonition} GROUP PROCESS CHECK-IN
+* How will we be making decisions?
+* Who decides what tasks will be worked on?
+* How will we navigate conflict?
+* How will we recognize / attribute effort?
+```
+
+(need some facilitation approach to ensure everyone is heard. 1-2-4 all?)
+
+### Step 3: Brainstorming Tasks
+
+Now that the team has had a chance to get to know each other and talk about your process and systems, it is time to begin identifying tasks. 
+
+```{admonition} CREATING A TASKBOARD
+* List the tasks that will need to be done (starting with any that the project leader may have already identified)
+* All team members are encouraged to contribute here. All ideas and perspectives are needed.
+* Identify any clusters of common tasks or tasks that must be completed sequentially
+* Team members volunteer for tasks that match their personal goals and interests, identify opportunities for working in pairs or small groups
+* Identify if any tasks require skills or knowledge that the current team does not yet have (the project team helper may be able to connect you with someone to provide those skills or knowledge)
```

Comment:
```suggestion
All team members are encouraged to contribute here. All ideas and perspectives are needed.
* List the tasks that will need to be done (starting with any that the project leader may have already identified).
* Identify any clusters of common tasks or tasks that must be completed sequentially.
* Team members volunteer for tasks that match their personal goals and interests, identify opportunities for working in pairs or small groups.
* Identify if any tasks require skills or knowledge that the current team does not yet have (the project team helper may be able to connect you with someone to provide those skills or knowledge).
```

Reply:
For clarity: this moves the note that everyone is encouraged to contribute to the top (instead of making it a task bullet point). It also adds periods.